src/app/zap-templates/zcl/data-model/silabs/zll.xml

```diff
@@ -53,7 +53,7 @@ limitations under the License.
     <item name="DecrementHue" value="0x00"/>
     <item name="IncrementHue" value="0x01"/>
   </enum>
-  <clusterExtension code="0x0000">
```

Comment:
This was referring to the basic cluster code from zigbee 3.0 which no longer exists in matter. With the new data type schema we run into an issue when we try to associate the attribute to a data type with a cluster code that does not exist. Therefore this is now associated with basic-information-cluster